tools/nnyeah/integration/API/Makefile

```diff
@@ -8,20 +8,35 @@ $(1)/bin/Debug/$(1)Integration.dll: $(wildcard *.cs */*.cs Makefile) $(1)/$(1)In
 	$(Q) $(SYSTEM_MSBUILD) $(1)/$(1)Integration.csproj $(MSBUILD_VERBOSITY)
 
 TEST_CLEAN_TARGETS += clean-$(1)
-clean-$(1):
+clean-$(1)::
 	$(Q) $(SYSTEM_MSBUILD) /t:Clean $(1)/$(1)Integration.csproj $(MSBUILD_VERBOSITY)
 	$(Q) rm -rf Converted/$(1)
 
 TEST_RUN_TARGETS += run-$(1)
-run-$(1): $(1)/bin/Debug/$(1)Integration.dll
+run-$(1):: $(1)/bin/Debug/$(1)Integration.dll
 	$(Q) rm -rf Converted/$(1)
 	$(Q) mkdir -p Converted/$(1)
 	$(Q) $(SYSTEM_DOTNET) run --project ../../nnyeah/nnyeah.csproj --input=$(1)/bin/Debug/$(1)Integration.dll --output=Converted/$(1)/$(1)Integration.dll --xamarin-assembly=$(2) --microsoft-assembly=$(3)
 
 endef
 
-$(eval $(call INTEGRATION_TEST_template,macOS,$(MAC_DESTDIR)/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/Xamarin.Mac/Xamarin.Mac.dll,$(DOTNET_DESTDIR)/Microsoft.macOS.Ref/ref/$(DOTNET_TFM)/Microsoft.macOS.dll))
-$(eval $(call INTEGRATION_TEST_template,iOS,$(IOS_DESTDIR)/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/Xamarin.iOS.dll,$(DOTNET_DESTDIR)/Microsoft.iOS.Ref/ref/$(DOTNET_TFM)/Microsoft.iOS.dll))
+LEGACY_IOS = $(IOS_DESTDIR)/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/Xamarin.iOS.dll
+NET_IOS = $(DOTNET_DESTDIR)/Microsoft.iOS.Ref/ref/$(DOTNET_TFM)/Microsoft.iOS.dll
+LEGACY_MAC = $(MAC_DESTDIR)/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/Xamarin.Mac/Xamarin.Mac.dll
+NET_MAC = $(DOTNET_DESTDIR)/Microsoft.macOS.Ref/ref/$(DOTNET_TFM)/Microsoft.macOS.dll
+
+$(eval $(call INTEGRATION_TEST_template,macOS,$(LEGACY_MAC),$(NET_MAC)))
+$(eval $(call INTEGRATION_TEST_template,iOS,$(LEGACY_IOS),$(NET_IOS)))
+
+nuget/.stamp-nuget-download:
```

Comment:
This looks good, but I'm guessing we're going to test more NuGets? If so, it might be better to (eventually) make this a C# unit test (C# is much easier to write than make code when things get more complex...)